apps/charterafrica/src/payload/collections/News.js

```diff
@@ -0,0 +1,18 @@
+import fields from "../fields/post";
+
+const News = {
+  slug: "news",
+  admin: {
+    useAsTitle: "title",
+    defaultColumns: ["title", "author"],
```

Comment:
Lets add a publication date here as well so that content editor can sort most recent posts, etc.